### ISSUE TITLE
Path Error in Ladybug Package 0.1.4 for Dynamo 1.2

### ISSUE THREAD

**Issue by Dheb19** [ladybug-analysis-tools/ladybug#17]
Hello,

Running the Ladybug Package 0.1.4 in Dynamo 1.2 gives me the following error on Ladybug nodes

Traceback (most recent call last):
File "<String>", line 45, in <module>
File "<String>", line 29, in tryGetPluginPath
IOError: [Errno 2] Could not find a part of the path 'C:\users\{user}\AppData\roaming\Dynamo\dynamo Core\1.2\DynamoSettings.xml' .: C:\users\{user}\AppData\roaming\Dynamo\dynamo Core\1.2\DynamoSettings.xml

**Comment by mostaphaRoudsari** [comment]
Hi @Dheb19, This is happening because ladybug can't find the Dynamo installation folder on your machine. Is your username `user{user}`?

Can you run this script in a Python node and let me know the output?
```python
from System import Environment
_appdata = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)
import sys
sys.path.append(r'C:\Program Files (x86)\IronPython 2.7\Lib')
import os

OUT = _appdata, os.getenv('APPDATA')
```

**Comment by Dheb19** [comment]
My user name is denis_h so the error is:

IOError: [Errno 2] Could not find a part of the path 'C:\users\denis_h\AppData\roaming\Dynamo\dynamo Core\1.2\DynamoSettings.xml' .: C:\users\denis_h\AppData\roaming\Dynamo\dynamo Core\1.2\DynamoSettings.xml

When I run the python script above, the output is "C:\Users\denis_h\AppData\Roaming" for both outputs

**Comment by mostaphaRoudsari** [comment]
Thanks! Let me check and get back to you soon. Seems to be the same issue here: https://github.com/ladybug-analysis-tools/honeybee-plus/issues/25#issuecomment-262285296

**Comment by mostaphaRoudsari** [comment]
@Dheb19 can you also check this reply: https://github.com/ladybug-analysis-tools/honeybee-plus/issues/25#issuecomment-262306942 and let me know the results. Thanks!

**Comment by mostaphaRoudsari** [comment]
This is fixed in version 0.1.5. Thank you for reporting the issue and your help to get it fixed!
